aptos-move/aptos-vm-types/src/resolver.rs

```diff
@@ -204,7 +219,8 @@ pub trait StateStorageView {
 /// resolve AggregatorV2 via the state-view based default implementation, as it
 /// doesn't provide a value exchange functionality).
 pub trait TExecutorView<K, T, L, V>:
-    TResourceView<Key = K, Layout = L>
+    BlockSynchronizationKillSwitch
```

Comment:
Actually, do we need it in executor view? You are using it via `impl AptosCodeStorage + BlockSynchronizationKillSwitch` so this should never bee needed? This might help with removing `?Sized`?

Reply:
good catch. removed, but didn't help there..

Reply:
:(

Reply:
it's solved now :( -> :)